### PR TITLE
JBPM-8421: Documents tab on Process Instance details doesn't show documents stored in documents variable ("org.jbpm.document.Documents")

### PR DESCRIPTION
**Pull request by pefernan**
@cristianonicolai @nmirasch @tomasdavidorg in order to make this development less painfull and reuse the existing functionality I had to index each individual document inside the documents list, so there will appear as many entries on the variables list as documents has the list. To identify them on the variables table, they have a specific name with the following format: "<var_name> (index/maxDocs)", so if a variable is storing 2 docs there will be two entries ("variable (1/2)" and "variable (2/2)").

See screenshots:
variables
variables_2

This is part of an ensemble, please merge with:
https://github.com/kiegroup/jbpm/pull/1480
https://github.com/kiegroup/jbpm-wb/pull/1334

### CODE REVIEW

**Comment by pefernan** [comment]
jenkins execute full downstream build

**Comment by pefernan** [comment]
jenkins execute full downstream build

File: jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/util/VariableHelper.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            VariableProcessor processor = processors.getOrDefault(value, new DefaultVariableProcessor());
+
+            processor.process(processInstanceId, key, value, filteredVariables, variableSummary -> {
+                variablesSummary.add(variableSummary);
+                variableSummary.setDeploymentId(deploymentId);
+                variableSummary.setServerTemplateId(serverTemplateId);
+            });
+        });
+
+        if(!filteredVariables.isEmpty()) {
```
[the remark this answers is not in the document]
Comment (pefernan) [reply jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/util/VariableHelper.java]: @tomasdavidorg thanks!

**Comment by nmirasch** [comment]
@jstastny-cz could you take a look? Thanks!

**Comment by pefernan** [comment]
jenkins execute full downstream build please

**Pull request merged**